.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,14 @@
+name: NPM Publish
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  npm-publish:
+    permissions:
+      contents: read
+      id-token: write
+    secrets: inherit
+    uses: hemilabs/actions/.github/workflows/npm-publish.yml@main
```

Comment:
We shall not use branch refs. Let's add version tags to hemilabs/actions and then go back to this PR. Or use sha refs so I can request for changes again.

Reply:
Thanks to Joshua, now it's using the tagged version [b25cf30](https://github.com/bloq/eth-packages/pull/9/commits/b25cf30374679af062b44af9043c584d1565e545)